back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -101,6 +124,35 @@ def resolve_tags(*_):
     return Tag.select()
 
 
+@query.field("packingListEntry")
+def resolve_packing_list_entry(*_, id):
+    authorize(permission="packing_list_entrie:read")
```

Comment:
This should be called `packing_list_entry`...maybe needs to be taken care of in our Auth0 script too